apps/modernization-ui/src/components/FormInputs/constant.ts

```diff
@@ -0,0 +1,40 @@
+const MONTH_LABELS = [
```

Comment:
suggestion:  Naming this file "constant" is going to tempt people to add unrelated things into it.  Since it only relates to the DatePicker component the content can be added to that file or renamed to be something more specific so that it remains related to the DatePicker localization only.